bin/alluxio-start.sh

```diff
@@ -276,6 +276,11 @@ start_master() {
       ALLUXIO_MASTER_JAVA_OPTS+=" -XX:MetaspaceSize=256M "
     fi
 
+    # use a CMS garbage collector for the master
+    if ! [[ "${ALLUXIO_MASTER_JAVA_OPTS}" =~ (UseConcMarkSweepGC|UseSerialGC|UseG1GC|UseParallelGC|UseParallelOldGC|UseParNewGC) ]]; then
```

Comment:
Maybe add something in our perf tuning docs about this? In case users want to use their own GC strategy, they can know to modify this line.

Reply:
Shouldn't they just use alluxio-env.sh to set this java opt - and then the conditional won't add the CMS?

Also, is this something better set within `./libexec/alluxio-config.sh`?

Reply:
Regarding the doc, I will push a separate PR for tuning GC strategies so the patch is not blocked on it.

Regarding where this logic should be, so far the `alluxio-config.sh` is more about Alluxio configuration properties while the `alluxio-env.sh` does the setting of JVM options. I will follow the convention for now but I am definitely open to discussion for potential refactor of the scripts.